src/style/dimension.rs

```diff
@@ -0,0 +1,67 @@
+//! A unit of linear measurement.
+
+#[cfg(feature = "std")]
+use super::CalcDimension;
+
+/// A unit of linear measurement
+///
+/// This is commonly combined with [`Rect`], [`Point`](crate::geometry::Point) and [`Size<T>`].
+/// The default value is [`Dimension::Undefined`].
+#[derive(Clone, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum Dimension {
```

Comment:
Should this have pixels too to start?

Reply:
@alice-i-cecile I believe that (contrary to the documentation), `Points` in taffy actually are pixels. I've been meaning to open an issue suggesting we rename the variant. Although I suppose, given that we only have one absolute unit, it could actually represent anything and it wouldn't make any difference. But for example, our generated tests use pixel values when running in chrome and Points in taffy.

Reply:
Ugh, that's annoying. We should rename it then.

Reply:
#236